library/alloc/src/string.rs

```diff
@@ -1849,6 +1849,35 @@ impl String {
         let slice = self.vec.into_boxed_slice();
         unsafe { from_boxed_utf8_unchecked(slice) }
     }
+
+    /// Consumes and leaks the `String`, returning a mutable reference to the contents,
+    /// `&'a mut str`.
```

Comment:
Nits: 'static

Reply:
> Nits: 'static

Oops, thanks for pointing this out!

Do you recommend I file a new PR right away, or bundle a fix in with a future stabilization PR?

Reply:
Filling new PR won't hurt if you want to do it, it will be rolled up.

Reply:
> Filling new PR won't hurt if you want to do it, it will be rolled up.

Done! https://github.com/rust-lang/rust/pull/103412